Domains/4-Application/Connectors/CivilInfrastructureFramework/CifRail.ecschema.xml

```diff
@@ -326,22 +388,33 @@
     </ECEntityClass>
     <ECEntityClass typeName="CantStationPointAspect" displayLabel="Cant Station Point">
         <BaseClass>cifcmn:CivilPresentation</BaseClass>
-        <ECProperty propertyName="CantStationPoint_TransitionType" typeName="string" displayLabel="Object Type" category="CantStationPoint_Cant_Station_Point" priority="299999"/>
-        <ECProperty propertyName="CantStationPoint_Station" typeName="double" displayLabel="Station" category="CantStationPoint_Cant_Station_Point" priority="299998" kindOfQuantity="cifu:STATION"/>
+        <ECProperty propertyName="CantStationPoint_TransitionType" typeName="CantStationPointAspect_CantStationPoint_TransitionType_Enum" displayLabel="Object Type" category="CantStationPoint_Cant_Station_Point" priority="299999"/>
+        <ECProperty propertyName="CantStationPoint_Station_Hidden" typeName="double" displayLabel="Station [Hidden]" category="CantStationPoint_Cant_Station_Point" priority="299998" kindOfQuantity="cifu:STATION">

```

Comment:
This property rename is considered a major schema upgrade that is not backwards compatible (i.e. delete + add). If you just meant to hide the property, it doesn't need to be renamed. Also, if the Connector won't write to it anymore, it is a good idea to mark it as deprecated as well (look for usage of the "Deprecated" custom-attribute in other Civil Connector schemas as an example).